test_closest (__main__.TestExercice) ... FAIL
test_coordinates (__main__.TestExercice) ... ok
test_linear_values (__main__.TestExercice) ... ERROR

======================================================================
ERROR: test_linear_values (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 17, in test_linear_values
    self.assertListEqual(
  File "/usr/lib/python3.8/unittest/case.py", line 1118, in assertListEqual
    self.assertSequenceEqual(list1, list2, msg, seq_type=list)
  File "/usr/lib/python3.8/unittest/case.py", line 1065, in assertSequenceEqual
    if item1 != item2:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

======================================================================
FAIL: test_closest (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 42, in test_closest
    self.assertEqual(
AssertionError: 10 != 3 : Erreur dans le remplacement de caract√®re

----------------------------------------------------------------------
Ran 3 tests in 0.007s

FAILED (failures=1, errors=1)
